module1.js

```diff
@@ -111,6 +133,10 @@ function question7(array1, array2, extraElement) {
 
 function question8(retailPrice, taxAmount) {
   // TODO:
+  var taxTotal = retailPrice * taxAmount;
+  var result = retailPrice.split();
+  //Not sure how to remove the currency symbol, tried to split the string and extract the currency symbol, store it in
```

Comment:
So looking at the test (and the question instruction), you can tell that `retailPrice` is actually a string, and not a number, so you can't use it for multiplication like you did on line 136.

What you were looking for to get the currency is `substring()` function on the String object - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substring.

So knowing that now, try to fix this function - the approach you wanted to take is correct :)